languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -24963,6 +24963,30 @@ USA
     <!-- Grammar -->
     <!-- ====================================================================== -->
     <category id="GRAMMAR" name="Grammar" type="grammar">
+        <rule id="QUALITY_PARTITIVE_SINGULAR" name="this kind of days (day)" default="temp_off">
+            <antipattern>
+                <token regexp="yes">this|that</token>
+                <token regexp="yes">kind|sort|type</token>
+                <token>of</token>
+                <token regexp="yes">stats|savings|epigenetics|genetics</token>
+                <example>This kind of stats is notoriously difficult.</example>
+                <example>That kind of savings could have mitigated the effect of the last rotating blackouts PG&amp;E had to impose.</example>
+            </antipattern>
+            <pattern>
+                <token regexp="yes">this|that</token>
+                <token regexp="yes">kind|sort|type</token>
+                <token>of</token>
+                <marker>
+                    <token postag="NNS">
+                    <exception postag="VBZ"/>
+                    <exception postag="NN:U"/>
+                    <exception postag="NN:UN"/>
```

Comment:
Only comment code-wise is that these exceptions can be done in one line:
`<exception postag="NN:UN?|VBZ" postag_regexp="yes" />`

The rule looks good!
I see that it's marked as a draft; I'll merge as soon as you mark it ready for review.